frontend_tests/node_tests/templates.js

```diff
@@ -129,6 +129,29 @@ function render(template_name, args) {
     assert.equal(emoji_url.attr('src'), 'http://emojipedia-us.s3.amazonaws.com/cache/46/7f/467fe69069c408e07517621f263ea9b5.png');
 }());
 
+(function admin_filter_list() {
+    // global.use_template('admin_filter_list');
```

Comment:
global.use_template no longer exists (the tracking is done automatically); you can delete this line entirely.